### ISSUE TITLE
RGCN generate nan in Pytorch 1.8

### ISSUE THREAD

**Issue by zheng-da** [dmlc/dgl#2875]
## 🐛 Bug

RGCN training generates nan.

```
ubuntu@ip-172-31-22-118:~/workspace/dgl/examples/pytorch/rgcn$ python3 entity_classify_mp.py -d ogbn-mag --testing --fanout='30,30' --batch-size 1024 --n-hidden 128 --lr 0.01 --num-worker 4 --eval-batch-size 8 --low-mem --gpu 0 --dropout 0.7 --use-self-loop --n-bases 2 --n-epochs 3 --node-feats --dgl-sparse --sparse-lr 0.08
Using backend: pytorch
WARNING:root:The OGB package is out of date. Your version is 1.2.2, while the latest version is 1.3.1.
Namespace(batch_size=1024, dataset='ogbn-mag', dgl_sparse=True, dropout=0.7, eval_batch_size=8, fanout='30,30', gpu='0', l2norm=0, layer_norm=False, low_mem=True, lr=0.01, n_bases=2, n_epochs=3, n_hidden=128, n_layers=2, node_feats=True, num_workers=4, sparse_lr=0.08, use_self_loop=True, validation=False)
Number of relations: 8
Number of class: 349
Number of train: 629571
Number of valid: 64879
Number of test: 41939
0:author
1:field_of_study
2:institution
3:paper
start training...
Train Accuracy: 0.0166 | Train Loss: 5.6925
Train Accuracy: 0.0117 | Train Loss: nan
Train Accuracy: 0.0029 | Train Loss: nan
Train Accuracy: 0.0029 | Train Loss: nan
Train Accuracy: 0.0049 | Train Loss: nan
Train Accuracy: 0.0049 | Train Loss: nan
Train Accuracy: 0.0029 | Train Loss: nan
Train Accuracy: 0.0029 | Train Loss: nan
```

## To Reproduce

1. install torch 1.8, dgl-cu101 0.6.1
1. go to `examples/pytorch/rgcn`
1. run `python3 entity_classify_mp.py -d ogbn-mag --testing --fanout='30,30' --batch-size 1024 --n-hidden 128 --lr 0.01 --num-worker 4 --eval-batch-size 8 --low-mem --gpu 0 --dropout 0.7 --use-self-loop --n-bases 2 --n-epochs 3 --node-feats --dgl-sparse --sparse-lr 0.08`

## Environment

 - DGL Version (e.g., 1.0): 0.6.1
 - Backend Library & Version (e.g., PyTorch 0.4.1, MXNet/Gluon 1.3): pytorch 1.8
 - OS (e.g., Linux): Ubuntu 18.04
 - How you installed DGL (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.6.9
 - CUDA/cuDNN version (if applicable): 10.1
 - GPU models and configuration (e.g. V100): rgcn
 - Any other relevant information:

**Comment by VoVAllen** [comment]
Is this duplicate with https://github.com/dmlc/dgl/issues/2760?

**Comment by zheng-da** [comment]
it seems so

**Comment by BarclayII** [comment]
Closing due to duplication.
